framework/resources/Functional/table_stats/stats/query/query.json

```diff
@@ -0,0 +1,31 @@
+{
+  "testId": "Table_Stats_Query",
```

Comment:
Test ID has to be unique. This is because all tests are collected and are identified as <testID>.<queryFileName>. I checked that your Test IDs here matches the one in the dir `positive`

Reply:
Oh ok I did not know that. I changed the test ID of positive json

Reply:
Thanks for updating!